setup.cfg

```diff
@@ -1,10 +1,10 @@
 [flake8]
```

Comment:
cc @aarnphm I think changes in this file should also trigger the style_check CI to run?

Reply:
ok we can add that into ci